benchmark/run_bench.sh

```diff
@@ -0,0 +1,207 @@
+#!/bin/bash
+
+rs=(1)
+cs=(1)
+w=10
+d=10
+reqs=(1)
+resps=(1)
+rpc_types=(unary)
+
+# idx[0] = idx value for rs
+# idx[1] = idx value for cs
+# idx[2] = idx value for reqs
+# idx[3] = idx value for resps
+# idx[4] = idx value for rpc_types
+idx=(0 0 0 0 0)
+idx_max=(1 1 1 1 1)
+
+inc()
+{
+  for i in $(seq $((${#idx[@]}-1)) -1 0)
```

Comment:
I prefer:

`for _____; do`

which you use below.  (Let's be consistent at the very least.)